fbw-a380x/src/base/flybywire-aircraft-a380-842/SimObjects/AirPlanes/FlyByWire_A380_842/panel/panel.cfg

```diff
@@ -151,5 +151,11 @@ htmlgauge00=WasmInstrument/WasmInstrument.html?wasm_module=systems.wasm&wasm_gau
 htmlgauge01=WasmInstrument/WasmInstrument.html?wasm_module=fbw.wasm&wasm_gauge=fbw, 0,0,1,1
 htmlgauge02=WasmInstrument/WasmInstrument.html?wasm_module=fadec.wasm&wasm_gauge=FadecGauge, 0,0,1,1
 htmlgauge03=WasmInstrument/WasmInstrument.html?wasm_module=extra-backend-a380x.wasm&wasm_gauge=Gauge_Extra_Backend,0,0,1,1
-# Uncomment when A380 ExtrasHost exists in monorepo
-#htmlgauge04=A380X/ExtrasHost/extras-host.html,0,0,1,1
+
+[VCockpit20]
```

Comment:
**question:** why create a new VCockpit entry for this?

Reply:
I don't think it belongs to the wasm stuff and also it creates a separate entry in the coherent debugger. 
But I don't care that much - but it doesn't hurt, does it?

Reply:
Each VCockpit has a bit of overhead. It would make sense to combine it with the other "headless" ones.

Reply:
> Each VCockpit has a bit of overhead. It would make sense to combine it with the other "headless" ones.

Good to know. Are there any advantages as well? Like better separation or even parallelization?

Still I think we should have systems-host and extras-host in one instrument separated from the wasm stuff. WASM is rather special and from my view this would make a bit more sense. 
Ok?